CHANGELOG.md

```diff
@@ -21,6 +21,7 @@ general bugfixes
 ## v2.5.0
 ###### DATA MIGRATIONS
 - Image icon_enabled is now False by default [#55](https://github.com/allink/allink-core/pull/55)
+- Removed onscreen effect in the allink_content plugin [#57](https://github.com/allink/allink-core/pull/57)
```

Comment:
there is no need to list this here. (also "Image icon_enabled is now False.." sorry, did't see it the last time.) we only list **data** Migrations here, which have to be done on project level manually. You can move them to NEW

Reply:
alright 👍